test/e2e/cypress/integration/checks_catalog.js

```diff
@@ -0,0 +1,54 @@
+import { availableChecks } from '../fixtures/checks-catalog/available_checks'
+
+context('Checks catalog', () => {
+    before(() => {
+        cy.resetDatabase()
+        cy.loadChecksCatalog('checks-catalog/catalog.json')
+
+        cy.visit('/catalog');
+        cy.url().should('include', '/catalog');
+    })
+
+    describe('Checks catalog should be available', () => {
+        it('should show 5 check groups in the catalog', () => {
+            cy.get('div.check-group').should('have.length', 5)
+        })
+        it('should show 35 checks in the catalog', () => {
+            cy.get('tr.check-row').should('have.length', 35)
+        })
+    })
+
+    describe('Checks grouping and identification is correct', () => {
+        availableChecks.forEach((checks, group) => {
+            it(`should include group '${group}'`, () => {
+                cy.get('.check-group > h4').should('contain', group)
+            })
+            checks.forEach((check_id) => {
+                it(`should include check '${check_id}'`, () => {
+                    cy.get('.check-row').should('contain', check_id)
+                })
+            })
+        })
+    })
+
+    describe('Individual checks data is expanded', () => {
+        it('should expand check data when clicked', () => {
+          const firstCheck = availableChecks.get('Corosync')[0]
+          cy.get('.check-row').filter(
```

Comment:
I am wondering if we need to get the `check-row` element and then filter by attrubute id.

We could just 
```
cy.get(`#${firstCheck}`).find('td > a.link-dark')
```

Or maybe change the html to add add a specific class or id to the `a` link element and simplify selectors.

What do you think?

Reply:
Yes, more or less it works. As removing some `filters` looks like reduces the time of these operations, the test started failing, so I have added other assertion (`have.class`) to wait until this is done.
This makes the test pass.
(change done in : https://github.com/trento-project/trento/pull/736/commits/4d3b70e79a16cbb37bfb1f235f937572f4ee1452)

Reply:
Great!